libexec/config-bootstrap-fixup

```diff
@@ -12,4 +12,8 @@ if [ -z "$MIRROR_HOST" ] || [ "$MIRROR_HOST" == "127.0.0.1" ]; then
   MIRROR_HOST=$GITIAN_HOST_IP
 fi
 
-sed "s;HOSTIP;$MIRROR_HOST;g" < target-bin/bootstrap-fixup.in > target-bin/bootstrap-fixup
+if [ $APT_CACHER = "1" ]; then
+    sed "s;HOSTIP;$MIRROR_HOST;g" < target-bin/bootstrap-fixup.in > target-bin/bootstrap-fixup
+else
+    sed "s;HOSTIP:3142/;;g" < target-bin/bootstrap-fixup.in > target-bin/bootstrap-fixup
+fi
```

Comment:
This is actually not working.

`config-boostrap-fixup` is called with `bin/make-base-vm`, which have `APT_CACHER` variable through `--disable-apt-cacher`.
It is called also with `bin/gbuild`, who use it with `make-clean-vm`. At this moment, they do not have `APT_CACHER` variable anymore.

Creating the following error :
```
bash: [: =: unary operator expected
```
`APT_CACHER` will be replaced by an unquoted empty value.

Reply:
As a possible solution, would it possible to configure [bootstrap-fixup.in](https://github.com/devrandom/gitian-builder/blob/master/target-bin/bootstrap-fixup.in) only when using `make-base-vm` ? And then reuse generated `bootstrap-fixup` file instead of recreating one in `make-clean-vm`.

The feature was implemented by commit https://github.com/devrandom/gitian-builder/commit/5785dfc3ae09aec37c4d9b8bc4b58410865e04fa, I suppose we can do it as I suggest in the case `GITIAN_HOST_IP` doesn't change between `make-base-vm` & `gbuild`.
Actually, without this situation, it looks like it's creating twice the same file.

It can make sure configuration are more done through `make-base-vm`.
Maybe deleting the following line can do the job (?) : 
https://github.com/devrandom/gitian-builder/blob/46093250577a646d71b20bde4bc007b3c5dc0d21/libexec/make-clean-vm#L65

Reply:
Not sure if it's the right solution, but if it is I just added commit https://github.com/devrandom/gitian-builder/pull/253/commits/d8d370de67a6d846c81cebb313af100572d9180f to implement this fix.